easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.0.8-iomkl-2018a.eb

```diff
@@ -0,0 +1,38 @@
+# Automatically converted from LibTIFF-4.0.8-intel-2017b.eb
+# Original message:
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>, Alan O'Cais (JSC)
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/
+##
+easyblock = 'ConfigureMake'
+
+name = 'LibTIFF'
+version = '4.0.8'
+
+homepage = 'http://www.remotesensing.org/libtiff/'
+description = "tiff: Library and tools for reading and writing TIFF data files"
+
+toolchain = {'name': 'iomkl', 'version': '2018a'}
```

Comment:
Existing easyconfig for LibTIFF 4.0.9 with `GCCcore/6.4.0` can be used to resolve dependency?